pinot-tools/src/main/java/org/apache/pinot/tools/PinotDataAndQueryAnonymizer.java

```diff
@@ -1070,6 +1132,25 @@ private String rewriteGroupBy(GroupByAstNode groupByAstNode) {
       return groupBy.toString();
     }
 
+    private String rewriteOrderBy(OrderByAstNode orderByAstNode) {
+      StringBuilder orderBy = new StringBuilder();
+      orderBy.append("ORDER BY ");
+      List<? extends AstNode> children = orderByAstNode.getChildren();
+      int count = 0;
+      for (AstNode orderByChild : children) {
+        if (count > 0) {
+          orderBy.append(", ");
+        }
+        Preconditions.checkState(orderByChild instanceof OrderByExpressionAstNode);
```

Comment:
we dont need the pre-condition. it is the same as throwing the class cast exception in the next line, right? (unless you are providing some meaningful exception string here)

Reply:
yes. removed